.changeset/afraid-mice-own.md

```diff
@@ -0,0 +1,8 @@
+---
+"@ledgerhq/wallet-api-simulator": minor
```

Comment:
We don't really need to bump the simulator for a new test

Reply:
Removed